Dockerfile.prod

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:20.04
+
+# Install dependencies and tini.
+RUN apt-get update \
+    && apt-get --no-install-recommends -y install libssl-dev libssl1.1 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Run as unprivileged user.
+RUN groupadd --system --gid 1001 nimiq \
+    && adduser --system --home /home/nimiq --uid 1001 --gid 1001 nimiq
+USER nimiq
+
+# Change homedir to nimiq
+WORKDIR /home/nimiq
+
+# Create nimiq directory for configuration
+RUN mkdir -p /home/nimiq/.nimiq
+
+# Set default config can be overwritten by mounting
+COPY ./client.toml /home/nimiq/.nimiq
+
+COPY ./target/release/nimiq-client /usr/local/bin/nimiq-client
+
+# Copy .ZKP to node.
+COPY ./.zkp /home/nimiq/.zkp/
+
+# Expose sync port
+EXPOSE 9100
+
+# Expose RPC port
+EXPOSE 8648
+
+# Expose metrics port
+EXPOSE 9200
+
+# Run CMD so we can use other bin
+CMD [ "nimiq-client" ]
```

Comment:
Use entrypoint except if you want other binaries to be accessable by overwriting the command